frontend/package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
```

Comment:
Why do we need all these upgrades?

Reply:
There are all there to add support for the Elvis operator.